### DISCUSSION TITLE
Integration with MLflow

### DISCUSSION THREAD

**Discussion by foxale** [great-expectations/great_expectations#5712] [Integrations]
**Is your feature request related to a problem? Please describe.**
Great expectations looks fantastic. We are thinking about adding it to our MLOps platform (Kedro, MLflow, etc.). It would make sens to have the validation checks tracked together with pipeline runs via MLFlow tracking, as currently the HTML reports are stored separately.

**Describe the solution you'd like**
I guess a great-mlflow plugin that would allow to link great-expectations with mlflow, so that the HTML validation reports can be opened from MLflow UI.

**Describe alternatives you've considered**
A separate great-expectations UI tracking validation runs that can be self-hosted somewhere (just like in MLflow).

**Comment by austiezr** [comment]
Hey @foxale! Thanks for raising this; sounds like great functionality, we'll review internally. Is this an area you'd be interested in making contributions in?

**Comment by foxale** [comment]
Absolutely :) I would imagine some basic, PoC-like functionality could be implemented with no changes on the mlflow side. However, for the more advanced integration they may actually be interested in a co-op.

**Comment by a-chumagin** [comment]
Hello! Kubeflow has integration with GX 
https://medium.com/analytics-vidhya/data-quality-assurance-with-great-expectations-and-kubeflow-pipelines-d83449fbaa81
